1230-doi_minting/1234-owner_drafts_a_doi_for_a_draft_publication.feature

```diff
@@ -0,0 +1,10 @@
+    @1234
+    Scenario: Owner drafts a DOI for a draft Publication
+        Given that an Owner views the Landing Page for their Publication
+        And the Publication has status Draft
+        And the Publication has no DOI
+        When they click the "Draft a DOI" button
+        Then the "Draft a DOI" button is renamed to "DOI pending" and is disabled
+        And the Landing Page for the Publication contains the Draft DOI
+        And the drafted DOI is not clickable
+# Draft DOIs are not acknowledged by the resolving mechanisms (Handle-system)

```

Comment:
hva er egentlig denne kommentaren her for? hva betyr den?

Reply:
at en draft doi i praksis er en "død" link som ikke peker på noe (enda)